lib/OpenQA/Utils.pm

```diff
@@ -1247,5 +1246,29 @@ sub set_listen_address {
     $ENV{MOJO_LISTEN} = join(',', @listen_addresses);
 }
 
+sub param_hash {
+    my ($controller, $param_name) = @_;
+
+    my $params = $controller->every_param($param_name) or return;
+    my %hash;
+    for my $param (@$params) {
+        # ignore empty params
+        next unless $param;
+        # allow passing multiple values by seaparating them with comma
```

Comment:
s/seaparating/separating/

Reply:
I thought this was a reference to the old testament :)